### ISSUE TITLE
Minor API change

### ISSUE THREAD

**Issue by zwei-beiner** [rdyro/torch2jax#13]
The neural net I'm using `torch2jax` for uses `torch.autograd` to compute gradients internally. To get `torch2jax` working, I had to delete the line `with torch.no_grad()` in `api.py` (2 occurrences) and in `gradients.py` (1 occurrence).

Would it be possible to make the use of `with torch.no_grad()` optional at the user-level? I suppose the only reason to use it here is for speedup during output shape inference. Maybe this could be something like `uses_gradients_internally` with default value `False`.

**Comment by rdyro** [comment]
Great catch! This is a bug, the no_grad option shouldn't be there at all. I've just updated it on the multi-gpu branch!

**Comment by zwei-beiner** [comment]
Thanks a lot for the quick response, closing this now.

**Comment by rdyro** [comment]
This is incorporated now!
